dbt/seeds/etl_fast_row_counts.csv

```diff
@@ -858,9 +858,9 @@ out_ferc1__yearly_pumped_storage_plants_sched408,2024,18
 out_ferc1__yearly_purchased_power_and_exchanges_sched326,2020,6717
 out_ferc1__yearly_purchased_power_and_exchanges_sched326,2021,6899
 out_ferc1__yearly_purchased_power_and_exchanges_sched326,2024,7065
-out_ferc1__yearly_rate_base,2020,13703
-out_ferc1__yearly_rate_base,2021,15033
-out_ferc1__yearly_rate_base,2024,15196
+out_ferc1__yearly_rate_base,2020,13699
+out_ferc1__yearly_rate_base,2021,15027
+out_ferc1__yearly_rate_base,2024,15190
```

Comment:
I don't understand why these row counts would have changed, given that @cmgosnell 's FERC 1 PR yesterday passed the fast CI on its way into the merge queue, and given that I didn't touch the FERC 1 data source at all in this PR.  I got these row counts by running `make pytest-coverage` locally and then resetting my `PUDL_OUTPUT` directory to point at the temporary pytest directory where the parquet outputs had landed.  I'm [running the CI in GHA now](https://github.com/catalyst-cooperative/pudl/actions/runs/14914960566/job/41898193084) to see if it comes to a different conclusion.

Reply:
That manual CI run failed, with 3 failures in this table so... even my local pytest outputs don't look the same as the ones being generated by the GHA CI.

Reply:
This feels related to our convo yesterday about discrepancies between local and remote CI runs.

Reply:
This one was especially baffling given that this PR touched nothing having to do with FERC 1 or the rate base table, and the previous fast row counts successfully made it into main and so had already passed the CI! I'm suspicious that something particularly weird is going on with the rate base table. But maybe it's just the same thing that's caused many mismatches between local pytest-coverage outputs and the CI outputs.